docs/install_guides/installing-openvino-brew.md

```diff
@@ -7,10 +7,13 @@
    Installing OpenVINO Runtime from Homebrew is recommended for C++ developers. 
    If you work with Python, consider :doc:`installing OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>`
 
-   The current Homebrew package provides inference support for CPU only.
+   Importantly, Homebrew always distributes the most recent package. You cannot use it to install previous versions of OpenVINO.
+    
+   The current Homebrew package provides inference support for CPU and GPU for Linux, but CPU only for macOS.
```

Comment:
CPU for Linux as well.
Via brew we provide all possible functionaity now:
- CPU on macOS x86_64, macOS arm64, Linux x86_64
- GPU on Linux x86_64, because we don't support Apple's GPU

Reply:
info included in an incoming PR